### PR TITLE
SW-4713 Update project meta fields (created by/time, modified by/time)

### PR DESCRIPTION
**Pull request by nickgraz**
- According to the latest design changes. 
- Fetch the meta data within the ParticipantProjectProvider so it is available in edit and single views.

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2425?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2426** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2426?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2425** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2425?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2424** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2424?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/ProjectField/Meta.tsx [hunk 1]
```diff
@@ -47,15 +50,20 @@ const ProjectFieldMeta = ({ date, dateLabel, user, userLabel }: ProjectFieldMeta
...
               marginBottom={theme.spacing(1)}
               component={'span'}
-              marginRight={theme.spacing(1)}
+              marginRight={theme.spacing(0.5)}
             >
               {userLabel}
             </Typography>
             {/* // TODO this will depend on what the BE returns, if we get a user object we can
             // do name and ID here but if they are spread into the project entity the structure
             // will be different */}
```
Comment (joshbuchea) [inline src/components/ProjectField/Meta.tsx:59]: Remove this comment?

**Review by joshbuchea — APPROVED** [review]
Looks good

**Comment by nickgraz** [comment]
### Merge activity

* **Apr 1, 11:12 AM EDT**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2425).

**Pull request merged**